dev-tool.sh

```
@@ -0,0 +1,124 @@
+#   - list:   List available modules.
+#   - choose: Select a module to work on (with tab-completion).
+#   - build:  Build the selected module.
+#   - start:  Run the selected module.
+#   - help: Show this help message.
+
+ACTIVE_MODULE=""
+OLD_PS1=$PS1
+
+# List available modules
+list() {
+  ls -d */ | grep "^[0-9][0-9]-" | sed 's/\///'
```

Shellcheck (0.10.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)